x-pack/plugins/fleet/public/components/uninstall_command_flyout/hooks.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+
+import { REDUCED_PLATFORM_OPTIONS } from '../../hooks';
+
+import type { Commands, UninstallCommandTarget } from './types';
+
+// todo: update with real API
+export const useCommands = (
```

Comment:
This should be renamed to useUninstalTokens or something similar as this is gonna fetch the uninstall token from the current policyId, the command will be build on the UI using this uninstall token right?

Reply:
yep, you're right! we will definitely need the token/command separation somewhere, either this will be a `useUninstallTokens`, or can stay the same and build the commands themselves

I'll merge it this way, and then when we have the API, I'll connect it to the API